src/scores/continuous/consistent_impl.py

```diff
@@ -0,0 +1,262 @@
+"""
+Implementation of scoring functions that are consistent for
+single-valued forecasts targeting quantiles, expectiles or Huber functionals.
+"""
+
+from typing import Callable, Optional
+
+import xarray as xr
+
+from scores.functions import apply_weights
+from scores.typing import FlexibleDimensionTypes
+from scores.utils import gather_dimensions
+
+
+def consistent_expectile_score(
+    fcst: xr.DataArray,
+    obs: xr.DataArray,
+    *,  # Force keywords arguments to be keyword-only
+    alpha: float,
+    phi: Callable[[xr.DataArray], xr.DataArray],
+    phi_prime: Callable[[xr.DataArray], xr.DataArray],
+    reduce_dims: Optional[FlexibleDimensionTypes] = None,
+    preserve_dims: Optional[FlexibleDimensionTypes] = None,
+    weights: Optional[xr.DataArray] = None,
+) -> xr.DataArray:
+    """
+    Returns the score using a scoring function that is consistent for the
```

Comment:
Maybe "Returns the generalised piecewise linear score which is consistent for ..."